app/views/certificates/_form.html.slim

```diff
@@ -17,6 +17,7 @@
       .row
         - Assignment::KINDS.each do |kind|
           .col-xs-2= fk.input kind, as: :boolean, input_html: { checked: @certificate.assignment_kinds[kind.to_s] }
+    = f.input :group_offer, as: :boolean, checked: @certificate.group_offer.to_s
```

Comment:
Hmm where does `@certificate.group_offer` come from? If I create a certificate in the console and try to call `group_offer` on it I get a `NoMethodError`.

Reply:
https://github.com/panter/aoz-003/blob/121c93124308efa6086b053e7e0f46a0deff634e/app/models/certificate.rb
in `certificate model` on line 12: 
```
def build_values
......
...group_offer: volunteer.group_offers.any?)
end
hmm.. but why doesn't it throw me an error in the views when one is created?

Reply:
Yeah looks like there are a few problems with the `Certificate` model :) Can you add some unit tests for the `build_values` behavior?

```ruby
[1] pry » c = Certificate.new
Creating scope :without_assignment. Overwriting existing method Volunteer.without_assignment.
NoMethodError: undefined method `hours' for nil:NilClass
from /home/mak/src/panter/aoz-003/app/models/certificate.rb:10:in `build_values'
```

```ruby
[2] pry » c = Certificate.new volunteer: Volunteer.first
...
ActiveModel::UnknownAttributeError: unknown attribute 'group_offer' for Certificate.
from /home/mak/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activemodel-5.1.4/lib/active_model/attribute_assignment.rb:48:in `_assign_attribute'
```

Reply:
Also, since the build passes we should probably add a feature spec to test certificate creation. I was trying to test it manually through the interface but couldn't figure out where to create them :)